contracts/modules/commons/ModuleAuth.sol

```diff
@@ -13,6 +13,9 @@ import "./ModuleERC165.sol";
 abstract contract ModuleAuth is IModuleAuth, ModuleERC165, SignatureValidator, IERC1271Wallet {
   using LibBytes for bytes;
 
+  uint256 private constant FLAG_SIGNATURE = 0;
+  uint256 private constant FLAG_ADDRESS = 1;
```

Comment:
Imo an Enum would be more elegant. e.g. `addressType.signature == 0x00` and `addressType.address == 0x01` or something. `signer` or `flag` is fine, whatever is clearer / more intuitive for you.

Reply:
I can benchmark it and see if the generated bytecode is any different, maybe one solution is cheaper than the other.